src/main/java/ladder/domain/Ladder.java

```diff
@@ -0,0 +1,44 @@
+package ladder.domain;
+
+import ladder.exception.LadderException;
+
+public class Ladder {
+
+    private static final String USER_AND_RESULT_COUNT_MATCH_ERR_MST =
+            "참가자와 게임 결과의 수가 일치해야 합니다.";
+
+    private final Lines lines;
+    private final LadderPrize ladderPrize;
+
+    public Ladder(Lines lines, LadderPrize ladderPrize) {
+        validateUserAndResult(lines.getLines().get(0).getNodes().size(), ladderPrize);
```

Comment:
```suggestion
        validateUserAndResult(lines.getStartPointCount(), ladderPrize);
```

디미터 원칙 위반입니다. 

classA.methodA의 최대 지식한계는 아래와 같습니다. 
- classA의 필드 객체
- methodA가 생성한 객체
- methodA의 인자로 넘어온 객체 

다른 객체 내부의 데이터가 무엇이 있는지까지 알아서는 안되며,
역할을 담당하고 있는 객체에 메시지를 보내서 응답받도록 작성해보세요.

https://developerfarm.wordpress.com/2012/01/30/object_calisthenics_5/